homeassistant/components/alexa/const.py

```diff
@@ -78,6 +78,9 @@
 API_THERMOSTAT_MODES_CUSTOM = {climate.HVAC_MODE_DRY: "DEHUMIDIFY"}
 API_THERMOSTAT_PRESETS = {climate.PRESET_ECO: "ECO"}
 
+# AlexaModeController does not like a single mode for the fan preset we add PRESET_MODE_NA if a fan has only one preset_mode
+PRESET_MODE_NA = "n/a"
```

Comment:
How about something like `Off`, `Disabled` or `None` ?

Reply:
The option 'n/a' is not perfoming any action. The frontend does not allow resetting the `preset_mode` using SERVICE_SET_PRESET_MODE. A `None` value will not reset the `preset_mode` this is a responsability for the backend. By filling the list with `None`, `Disabled` or 'Off' the user might be given the suggestion something can be selected here.
Perhaps just a `-` could be even clearer.

Reply:
`-` would more locale agnostic. 🤷

Reply:
Done, change to`-` the user will get a neat message in the Alexa App that the device does not support the select preset.